uperf/uperf_run

```diff
@@ -114,7 +114,7 @@ usage()
 	echo "  --packet_type:  comma separated list of packet types, (tcp,udp)"
 	echo "  --server_ips: comma separated list of the server ip addresses"
 	echo "  --suffix: Suffix to add to the results file"
-	echo "  --test_types:  stream,rr,maerts,bidirec"
+	echo "  --tests:  stream,rr,maerts,bidirec"
```

Comment:
Is this necessary for the results fix?

Reply:
It is just changing the usage to what it should be.  If you want to be nitty we can open a separate pr and make the the change there.

Reply:
As long as it's separate commits I'm not going to jump off a cliff over this one.  Just one rap on the knuckles with a ruler this time.

Reply:
This has been moved to its own pr.